style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,200&display=swap');
+
+.home {
+  display: grid;
+  grid-template-rows: 92px 810px;
+  width: 375px;
+}
+
+.header {
+  background-color: #3c3a39;
+}
+
+.header h1 {
+  color: #fff;
+  position: relative;
+  margin-top: 56px;
+  margin-bottom: 12px;
+  margin-left: 24px;
+  font-size: 18px;
+}
+
+.header a {
+  position: absolute;
```

Comment:
We need to lay out the elements with the use of Flexbox and not using relative or absolute positioning;

- [ ] Please reconsider restyling the elements and remove the absolute positioning of this element; :100: